packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json

```diff
@@ -0,0 +1,1186 @@
+            "git-error": "erro de git",
+            "con-failed": "Conexão falhou",
+            "not-git": "Não é um repositório git",
+            "no-resource": "Repositório não encontrado",
+            "cant-get-ssh-key-path": "Erro! Não é possível obter o caminho da chave SSH selecionado.",
+            "unexpected_error": "erro_inesperado",
+            "clearContext": "Limpar contexto quando ocorrer troca de projetos"
+        },
+        "delete": {
+            "confirm": "Tem certeza de que deseja excluir este projeto?"
+        },
+        "create-project-list": {
+            "search": "procure seus projetos",
+            "current": "atual"
+        },
+        "require-clean": {
+            "confirm": "<p>Você tem alterações não implantadas que serão perdidas. </p> <p> Deseja continuar?</p>"
+        },
+        "send-req": {
+            "auth-req": "Autenticação necessária para repositório",
+            "username": "Nome do usuário",
+            "password": "Senha",
+            "passphrase": "Frase de Passe",
+            "retry": "Tentar novamente",
+            "update-failed": "Falha ao atualizar autenticação",
+            "unhandled": "Resposta de erro não tratada",
+            "host-key-verify-failed": "<p>Falha na verificação da chave do servidor anfitrião. </p> <p> A chave do servidor anfitrião do repositório não pôde ser verificada. Atualize seu arquivo <code> known_hosts </code> e tente novamente.</p>"
+        },
+        "create-branch-list": {
+            "invalid": "Ramo inválido",
+            "create": "Criar ramo",
+            "current": "atual"
+        },
+        "create-default-file-set": {
+            "no-active": "Não é possível criar um conjunto de arquivos padrão sem um projeto ativo",
+            "no-empty": "Não é possível criar um arquivo padrão definido em um projeto não vazio",
+            "git-error": "erro no git"
+        },
+        "errors": {
+            "no-username-email": "Seu cliente Git não está configurado com um nome de usuário / e-mail.",
+            "unexpected": "Um erro inesperado ocorreu",
+            "code": "código"
+        }
+    },
+    "editor-tab": {
+        "properties": "Propriedades",
+        "envProperties": "Variáveis de Ambiente",
+        "module": "Propriedades do Módulo",
+        "description": "Descrição",
+        "appearance": "Aparência",
+        "preview": "Visualização da IU",
+        "defaultValue": "Valor padrão"
+    },
+    "tourGuide": {
+        "takeATour": "Faça um tour",
+        "start": "Inicio",
+        "next": "Próximo",
+        "welcomeTours": "Tour de Boas-vindas"
+    },
+    "diagnostics": {
+        "title": "informações do Sistema"
+    },
+    "languages": {
+        "de": "Alemão",
+        "en-US": "Inglês",
+        "ja": "Japonês",
+        "ko": "Coreano",
+        "pt-BR": "Português(Brasil)",
+        "ru": "Russo",
+        "zh-CN": "Chinês(Simplificado)",
+        "zh-TW": "Chinês(Tradicional)"
+     }
```

Comment:
There are two sections missing here: `validator` and `contextMenu`

Reply:
Thanks for paying attention to the details Mauricio. Good Job, Already fixed!